### DISCUSSION TITLE
The issue of unclear full screen display at 4K resolution.

### DISCUSSION THREAD

**Discussion by Jason0247** [airsdk/Adobe-Runtime-Support#2716] [Q&A] [answered]
When I use a 4K resolution, the zoom and layout settings for Windows are 150%. However, when I run the program, the resolution is not 4K, and I need to change the 150% to 100% to display it clearly.

May I ask if there are any settings for AIR that can run the display in 4K mode during runtime? Instead of manually adjusting the zoom and layout of Windows to display the content clearly?
Thank you!

**Comment by 2jfw** [comment, marked as answer]
I think you just need to add

`		<requestedDisplayResolution>high</requestedDisplayResolution>`

in your `Application Descriptor` (Main-app.xml) under the _initialWindow_-Tag

**Comment by Jason0247** [comment]
I just discovered a problem, maybe I don't understand.
When I load a bitmap and set the Windows resolution to 100%, the display is not jagged.
![image](https://github.com/airsdk/Adobe-Runtime-Support/assets/91530077/848c4eae-4cfb-4b6b-b663-705089ee0b78)
![image](https://github.com/airsdk/Adobe-Runtime-Support/assets/91530077/f91840ab-a7de-4a6f-acde-04b389ac1424)
![image](https://github.com/airsdk/Adobe-Runtime-Support/assets/91530077/41a42703-41c6-48c6-b423-a9613d1c71ea)
![image](https://github.com/airsdk/Adobe-Runtime-Support/assets/91530077/13f9df75-83fd-4e14-8b07-007807d8ffe8)


When I set it to 150%, this image is jagged.
I have set<requestedDisplayResolution>high</requestedDisplayResolution>in the descriptor
When the scaling and layout of Windows are at 150% (recommended), when I output flash. system. Capabilities. screenResolutionX, 2560 is still displayed, and the content of the bitmap will be blurred. Only by setting the scaling ratio of Windows to 100% can I solve this problem.
I want to know what the meaning is when the requestedDisplayResolution tag is set to high? Because it seems to be essentially the same.
During initialization, I set the following code:
Stage. align="TL";
Stage. scaleMode="noScale";
Stage. displayState=flash. display. StageDisplayState. Full_ SCREEN_ INTERACTIVE;
